.github/workflows/rw-publish-prerelease.yml

```diff
@@ -141,3 +141,28 @@ jobs:
           RELEASE_VERSION: ${{ needs.bump-version.outputs.version }}
           SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
           PRERELEASE: ${{ needs.setup-params.outputs.prerelease-tag }}
+
+  notify-gdc-ui:
+    if: ${{  needs.setup-params.outputs.prerelease-id == 'alpha' }}
+    runs-on:
+      group: infra1-runners-arc
+      labels: runners-small
+    needs: [publish-prerelease,setup-params]
```

Comment:
do we need setup-params which already is a dependency of publish-prerelease here? (I don't see we would consume any of its outputs)

Reply:
we need it becouse I need do check if we are publishing alpha and thats setup-params job

Reply:
ah, I see now :-)